src/plugins/intel_cpu/src/transformations/transformation_pipeline.h

```diff
@@ -41,6 +39,7 @@ class Transformations {
 
     void UpToLpt();
     void CpuSpecificOpSet();
+    void RunPrecisionConvert();
```

Comment:
Could you please point out on where is this method used?

Reply:
It is old solution for precision unsupported issue, now can remove it.